rules/hmap.bzl

```diff
@@ -74,14 +74,23 @@ def _make_headermap_impl(ctx):
     )
     cc_info_provider = CcInfo(compilation_context = objc_provider.compilation_context)
 
-    return [
-        HeaderMapInfo(
-            files = depset([ctx.outputs.headermap]),
-        ),
+    contains_headers = False
+    for h in hdrs_lists:
```

Comment:
`hdrs_lists =[l for l in hdrs_lists if l]`

And then the conditional boils down to `if hdrs_lists:` ?